source/_patterns/00-atoms/components/speech-bubble.yaml

```diff
@@ -0,0 +1,37 @@
+assets:
+  css:
+    - speech-bubble.css
+    - ../../../../assets/sass/patterns/molecules/hypothesis-opener.css
```

Comment:
Doesn't look right?

Reply:
Yeah, I'm not very comfortable with this. It's like this because HypothesisOpener is now not a pattern, but still requires some positioning: this is calling in the css file for that directly rather than referring to a `yaml` file (because now there isn't one). Considered applying the style through JavaScript and so losing `hypothesis-opener.css` completely, the but that felt wrong. On reflection I think I might go for that option though, I think this is wronger.